internal/distro/rhel85/partition_tables.go

```diff
@@ -324,6 +238,86 @@ func createPartition(
 	}
 }
 
+func createRootPartition(start uint64, size uint64, archName string, uuid string) disk.Partition {
+	if archName == "x86_64" || archName == "aarch64" {
+		return disk.Partition{
+			Start: start,
+			Size:  size,
+			Type:  "0FC63DAF-8483-4772-8E79-3D69D8477DE4",
+			UUID:  "6264D520-3FB9-423F-8AB8-7A0A8E3D3562",
+			Filesystem: &disk.Filesystem{
+				Type:         "xfs",
+				UUID:         uuid,
+				Label:        "root",
+				Mountpoint:   "/",
+				FSTabOptions: "defaults",
+				FSTabFreq:    0,
+				FSTabPassNo:  0,
+			},
+		}
+	}
+	if archName == "ppc64le" {
+		return disk.Partition{
+			Start: start,
+			Size:  size,
+			Filesystem: &disk.Filesystem{
+				Type:         "xfs",
+				UUID:         uuid,
+				Mountpoint:   "/",
+				FSTabOptions: "defaults",
+				FSTabFreq:    0,
+				FSTabPassNo:  0,
+			},
+		}
+	}
+	return disk.Partition{
+		Start:    start,
+		Size:     size,
+		Bootable: true,
+		Filesystem: &disk.Filesystem{
+			Type:         "xfs",
+			UUID:         uuid,
+			Mountpoint:   "/",
+			FSTabOptions: "defaults",
+			FSTabFreq:    0,
+			FSTabPassNo:  0,
+		},
+	}
+}
+
+func createBootPartition(mountpoint string, start uint64, size uint64) disk.Partition {
+	if mountpoint == "/boot/efi" {
+		return disk.Partition{
+			Start: start,
+			Size:  size,
+			Type:  "C12A7328-F81F-11D2-BA4B-00A0C93EC93B",
+			UUID:  "68B2905B-DF3E-4FB3-80FA-49D1E773AA33",
+			Filesystem: &disk.Filesystem{
+				Type:         "vfat",
+				UUID:         "7B77-95E7",
+				Mountpoint:   "/boot/efi",
+				FSTabOptions: "defaults,uid=0,gid=0,umask=077,shortname=winnt",
+				FSTabFreq:    0,
+				FSTabPassNo:  2,
+			},
+		}
+	}
+	return disk.Partition{
+		Start: start,
+		Size:  size,
+		Type:  "C12A7328-F81F-11D2-BA4B-00A0C93EC93B",
+		UUID:  "68B2905B-DF3E-4FB3-80FA-49D1E773AA33",
+		Filesystem: &disk.Filesystem{
+			Type:         "xfs",
+			UUID:         "2AE383D1-F57C-4F04-A1BD-32FC12A578EA",
```

Comment:
Just a minor note: xfs partitions must have a random uuid because there's a global table in kernel.

Reply:
I had no idea. Thanks for that, I'll update it.